packages/lesswrong/components/vulcan-forms/Form.tsx

```diff
@@ -137,7 +137,7 @@ const getInitialStateFromProps = nextProps => {
 /**
  * Note: Only use this through WrappedSmartForm
  */
-class SmartForm extends Component<any,any> {
+class Form extends Component<any,any> {
```

Comment:
When pairing with JP, we both got confused by the naming here and in `FormWrapper.tsx` below. So we decided to clean up the naming in a few form-related files.